### DISCUSSION TITLE
hardware for cp-sat solver

### DISCUSSION THREAD

**Discussion by gregy4** [google/or-tools#3489] [CP-SAT questions]
Hello,
  since hedt computers are dead, gap between most powerful consumer cpus like amd 5950x or new 7950x and workstation cpus like threadripper pro is too big. Main difference is max memory, limited currently to 128gb on both intel and amd consumer platforms, and memory bandwidth limited to two channels compared to eight for threadripper pro.

  I'm thinking about new amd 7950x but don't know whether for big models using over 1million variables, over 70gb ram and all cores memory bandwith will not be main limitation. Is consumer cpu sufficient for big models or much pricier alternative like threadripper pro is the way to go ?

Thanks,
Jan

**Comment by lperron** [comment]
16 real cores is good.
I would go with the newer one.

I have not much to add :-)
